index.html

```diff
@@ -7,33 +7,241 @@
-          Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work.
-        </p>
-        <p class="intro-p dark-blue">
-          If you like what you see and have a project you need coded, don't hesitate to contact me.
-        </p>
-        <button class="start-collab bg-dark-blue">Start collaboration</button>
+  <!--Headline section-->
+  <div>
+    <header>
+      <nav class="top-edge-container">
+        <div class="geo-background-img"></div>
+        <div class="ic-menu"></div>
+      </nav>
+    </header>
+    <section class="section-container">
+      <div class="section-sub">
+        <h1 class="title dark-blue">Charlie Giorgania</h1>
+        <div class="contents">
+          <p class="paragraph dark-blue">
+            Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work.
+          </p>
+          <p class="paragraph dark-blue">
+            If you like what you see and have a project you need coded, don't hesitate to contact me.
+          </p>
+          <button class="collab-button bg-dark-blue white">Start collaboration</button>
+        </div>
+      </div>
+      <div>
+        <div class="thur-background-img css-background-img"></div>
       </div>
+    </section>
+  </div>
+  
+  <!--Works container 1-->
+  <secion class="section-container">
+    <div class="section-sub">
+      <div><h1 class="title dark-blue">Projects</h1></div>
+    </div>
+
+    <div>
+      <img class="img-laptop1" src="images/laptop-img.png" alt="This is an image of a person using a laptop." width="320" height="186">
+    </div>
+    <div class="section-dark bg-dark-blue">
+      <p class="project-goes white">Project name goes here</p>
+      <ul class="space">
+        <li class="greenish-box">HTML/CSS</li>
+        <li class="greenish-box">Ruby on Rails</li>
+        <li class="greenish-box">JavaScript</li>
+      </ul>
+      <button class="see-button bg-dark-blue white">
+        <div class="pointing">
+          <p>See this project</p>
+          <img class="icon" src="images/arrow.png" alt="This is a arrow pointing to the right icon" width="21" height="16">
+        </div>
+      </button>
+    </div>
+    
+    <div>
+      <img class="img-laptop1" src="images/laptop-img2.png" alt="This is an image of a laptop." width="320" height="186">
+    </div>
+    <div class="section-dark bg-dark-blue">
+      <p class="project-goes white">Project name goes here</p>
+      <ul class="space">
+        <li class="greenish-box">HTML/CSS</li>
+        <li class="greenish-box">Ruby on Rails</li>
+        <li class="greenish-box">JavaScript</li>
+      </ul>
+      <button class="see-button bg-dark-blue white">
+        <div class="pointing">
+          <p>See this project</p>
+          <img class="icon" src="images/arrow.png" alt="This is a arrow pointing to the right icon" width="21" height="16">
+        </div>
+      </button>
     </div>
```

Comment:
- [ ] To make your project **more aligned with best practices and more responsive**, I suggest adding both your image and project details inside a parent **div** and aligning them using `display: flexbox`. **This is also a project requirement that you need to fulfill**.

  Using `display: flexbox` can help you easily align and position both elements side by side, while maintaining their responsiveness on different screen sizes. **This is a best practice to follow, as it can enhance the overall user experience of your project and make it look more professional**. 🤓
  
  ----
  ### Project requirement:
  ![image](https://user-images.githubusercontent.com/107952724/236647251-80d4f230-5089-434e-a3a3-ca13dd730bba.png)